jest.config.js

```diff
@@ -0,0 +1,16 @@
+// Taken from ChatGPT
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'jsdom', // Use 'jsdom' for JSX rendering support
+  transform: {
+    '^.+\\.js$': 'babel-jest', // Transforms JavaScript files with babel-jest
+    '^.+\\.tsx?$': 'ts-jest', // Handles .ts and .tsx files
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'], // Ensure it recognizes .tsx files
+  testPathIgnorePatterns: ['/cypress/'],
+  moduleNameMapper: {
+    // Mock CSS imports, suggested by ChatGPT
+    '\\.(css|less|scss|sass)$': '<rootDir>/src/test/style-mock.js',
```

Comment:
CC projects generally use the `identity-obj-proxy` for this:
```suggestion
    "\\.(css|less|sass|scss)$": "identity-obj-proxy",
```
but I suspect it amounts to the same thing in the end. It's just a means of telling jest to ignore styles since they're irrelevant.